notebooks/official/feature_store/sdk_feature_store.ipynb

```diff
@@ -0,0 +1,2418 @@
+{
```

Comment:
<p>Before you check in, make sure the notebook output is cleared. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Should be cleared out now.